tableauserverclient/server/server.py

```diff
@@ -103,6 +103,8 @@ def __init__(self, server_address, use_server_version=False, http_options=None,
 
     def validate_server_connection(self):
         try:
+            if not self._server_address.startswith("http"):
+                self._server_address = "http://" + self._server_address
```

Comment:
Seems weird to default to `http` in 2022.

Reply:
yep - maintaining backward compatibility for tabcmd